triphouse/src/components/HotelsItem.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from 'prop-types';
+
+function HotelsItem(props){
+  HotelsItem.propTypes = {
+    hotel: PropTypes.object,
+  };
+  const { city, imageUrl, name, country } = props.hotel;
+  return(
+        <div className='homes-item' >
+          <img className="homes-img" src={imageUrl} alt={name} />
+          <a className="homes-link" href="#">{name}</a>
+          <span className="homes-place">{city}, {country}</span>
+        </div>
+      );
+}
+
+export default HotelsItem;
```

Comment:
store propTypes after the function. it is much cleaner and more convenient to read
![image](https://user-images.githubusercontent.com/20189313/143792168-776a4ce2-9895-45ea-89a4-5b277fc45b8c.png)